Sources/Apodini/Information/InformationClass.swift

```diff
@@ -0,0 +1,54 @@
+//
+// Created by Andreas Bauer on 06.07.21.
+//
+
+/// A ```InformationClass``` is a way to group multiple, similar ``Information`` types
+/// which represent the same Metadata information and can be mapped to the same type of
+/// key and value.
+/// A ``InformationClass`` is typically defined for a specific ``InterfaceExporter`` or a group
```

Comment:
Not sure, but I think you have to add a blank line here to start the DocC Discussion section.